frontend/src/functions/filterAliases.test.ts

```diff
@@ -100,7 +100,7 @@ it("filters out aliases that do not match the given string", () => {
 it("can match on an alias's subdomain when using a string filter", () => {
   const aliases = [
     getMockedAliasMatching({ domainType: "random" }),
-    getMockedAliasMatching({ domainType: "custom" }),
+    getMockedAliasMatching({ domainType: "custom" }, "some_subdomain"),
   ];
 
   expect(filterAliases(aliases, { string: "some" })).toStrictEqual([
```

Comment:
This `"some"` now refers to the `some` on line 103, rather than the one that happened to be set on line 11, outside of this test.

Reply:
Weird the inline diffe says line 103, but I think you meant line 125? In any case, looks good and tests still pass! Ship it!